acme/acme/client.py

```diff
@@ -1171,7 +1163,7 @@ def _add_nonce(self, response):
         if self.REPLAY_NONCE_HEADER in response.headers:
             nonce = response.headers[self.REPLAY_NONCE_HEADER]
             try:
-                decoded_nonce = jws.Header._fields['nonce'].decode(nonce)
+                decoded_nonce = jws.Header._fields['nonce'].decode(nonce)  # pylint: disable=no-member
```

Comment:
I think this `pylint: disable` can be deleted.

Reply:
Done.